### PR TITLE
[FSDE-28735] Add Tabbed Navigation in Chi documentation

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

**Review by mattnickles — COMMENTED** [review]
@jllr @josecarlosgonzalezv @soujanreddy Drop down tab items appear to be missing [a chevron icon](https://assets.ctl.io/chi/5.24.0/components/tabs/?theme=Lumen#tabbed-navigation-bar). Also it looks like we lost the ability to open drop down tab items with the keyboard - need this to pass accessibility.

**Review by mattnickles — COMMENTED** [review]
@jllr @josecarlosgonzalezv @soujanreddy In the previous implementation, we preserved the -active class on the parent tab item li if one of its children was set to active. The parent no longer renders with an -active class which is causing the text color to display as black instead of blue.

image

File: src/website/views/components/tabs/_examples.pug [hunk 1]
```diff
@@ -1075,46 +1075,151 @@ h2 Examples
...
-          a(href=`#webcomponent-tabbed-navigation`) Web Component
         li.-active
+          a(href=`#webcomponent-tabbed-navigation`) Web Component
+        li
           a(href=`#html-tabbed-navigation`) HTML Blueprint
-    .chi-tabs-panel#webcomponent-tabbed-navigation(role="tabpanel")
-    .chi-tabs-panel.-active#html-tabbed-navigation(role="tabpanel")
+    .chi-tabs-panel.-active#webcomponent-tabbed-navigation(role="tabpanel")
+      :code(lang='html')
+        <chi-tabs active-tab="tab-a" id="example__tabbednavigation_flat" sliding-border></chi-tabs>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1087]:
```suggestion
        <chi-tabs active-tab="tab-a" id="example__tabbed_navigation_flat" sliding-border></chi-tabs>
```

File: src/website/views/components/tabs/_examples.pug [hunk 2]
```diff
@@ -1075,46 +1075,151 @@ h2 Examples
...
+
+        <script>
+          const tabbedNavigationFlatElement = document.querySelector('#example__tabbednavigation_flat');
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1090]:
```suggestion
          const tabbedNavigationFlatElement = document.querySelector('#example__tabbed_navigation_flat');
```

File: src/website/views/components/tabs/_examples.pug [hunk 3]
```diff
@@ -1075,46 +1075,151 @@ h2 Examples
...
+          tabbedNavigationFlatElement.tabs = [
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1091]:
```suggestion
          
          tabbedNavigationFlatElement.tabs = [
```

File: src/website/views/components/tabs/_examples.pug [hunk 4]
```diff
@@ -1172,42 +1277,143 @@ h2 Examples
...
+        li
           a(href=`#html-solid-tabbed-navigation`) HTML Blueprint
-    .chi-tabs-panel#webcomponent-solid-tabbed-navigation(role="tabpanel")
-    .chi-tabs-panel.-active#html-solid-tabbed-navigation(role="tabpanel")
+    .chi-tabs-panel.-active#webcomponent-solid-tabbed-navigation(role="tabpanel")
+      :code(lang='html')
+        <chi-tabs active-tab="tab-aa" id="example__tabbednavigation_solid" solid sliding-border></chi-tabs>
+
+        <script>
+          const tabbedNavigationSolidElement = document.querySelector('#example__tabbednavigation_solid');
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1292]:
```suggestion
          const tabbedNavigationSolidElement = document.querySelector('#example__tabbed_navigation_solid');
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/website/views/components/tabs/_examples.pug [hunk 5]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
-          a(href='#exampleHashTarget') Tab Link
-        li
-          a(href='#exampleHashTarget') Tab Link
-        li.chi-dropdown
-          a.chi-dropdown__trigger(href='#') Tab Dropdown
-          .chi-dropdown__menu
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 1
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 2
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 3
+      chi-tabs(active-tab='tab-a' id='example_tabbed_navigation_flat' sliding-border)
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1078]:
```suggestion
      chi-tabs(active-tab='tab-a' id='example__tabbed_navigation_flat' sliding-border)
```

File: src/website/views/components/tabs/_examples.pug [hunk 6]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
-          a(href='#exampleHashTarget') Tab Link
-        li
-          a(href='#exampleHashTarget') Tab Link
-        li.chi-dropdown
-          a.chi-dropdown__trigger(href='#') Tab Dropdown
-          .chi-dropdown__menu
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 1
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 2
-            a.chi-dropdown__menu-item(href='#exampleHashTarget') Element 3
+      chi-tabs(active-tab='tab-aa' id='example_tabbed_navigation_solid' solid sliding-border)
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1281]:
```suggestion
      chi-tabs(active-tab='tab-aa' id='example__tabbed_navigation_solid' solid sliding-border)
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

File: src/website/views/components/tabs/_examples.pug [hunk 7]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                  label: 'Element 1',
+                  id: 'tab-h-1'
+                },
+                {
+                  label: 'Element 2',
+                  id: 'tab-h-2',
+                  children: [
+                    {
+                      label: 'Elem 1',
+                      id: 'tab-h-1'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1185]:
```suggestion
                      id: 'tab-h-2-1'
```

File: src/website/views/components/tabs/_examples.pug [hunk 8]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                      label: 'Elem 2',
+                      id: 'tab-h-2'
+                    },
+                    {
+                      label: 'Elem 3 more',
+                      id: 'tab-h-3',
+                      children: [
+                        {
+                          label: 'Elem 3.1',
+                          id: 'tab-h-3-1'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1197]:
```suggestion
                          id: 'tab-h-2-3-1'
```

File: src/website/views/components/tabs/_examples.pug [hunk 9]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                        },
+                        {
+                          label: 'Elem 3.2',
+                          id: 'tab-h-3-2'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1201]:
```suggestion
                          id: 'tab-h-2-3-2'
```

File: src/website/views/components/tabs/_examples.pug [hunk 10]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                        },
+                        {
+                          label: 'Elem 3.3',
+                          id: 'tab-h-3-3'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1205]:
```suggestion
                          id: 'tab-h-2-3-3'
```

File: src/website/views/components/tabs/_examples.pug [hunk 11]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                        }
+                      ]
+                    },
+                    {
+                      label: 'Elem 4',
+                      id: 'tab-h-4'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1211]:
```suggestion
                      id: 'tab-h-2-4'
```

File: src/website/views/components/tabs/_examples.pug [hunk 12]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                  label: 'Element 1',
+                  id: 'tab-f-1'
+                },
+                {
+                  label: 'Element 2',
+                  id: 'tab-f-2',
+                  children: [
+                    {
+                      label: 'Elem 1',
+                      id: 'tab-f-1'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1380]:
```suggestion
                      id: 'tab-f-2-1'
```

File: src/website/views/components/tabs/_examples.pug [hunk 13]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                    },
+                    {
+                      label: 'Elem 2',
+                      id: 'tab-f-2'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1384]:
```suggestion
                      id: 'tab-f-2-2'
```

File: src/website/views/components/tabs/_examples.pug [hunk 14]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                    },
+                    {
+                      label: 'Elem 3 more',
+                      id: 'tab-f-3',
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1388]:
```suggestion
                      id: 'tab-f-2-3',
```

File: src/website/views/components/tabs/_examples.pug [hunk 15]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                      children: [
+                        {
+                          label: 'Elem 3.1',
+                          id: 'tab-f-3-1'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1392]:
```suggestion
                          id: 'tab-f-2-3-1'
```

File: src/website/views/components/tabs/_examples.pug [hunk 16]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                        },
+                        {
+                          label: 'Elem 3.2',
+                          id: 'tab-f-3-2'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1396]:
```suggestion
                          id: 'tab-f-2-3-2'
```

File: src/website/views/components/tabs/_examples.pug [hunk 17]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                        },
+                        {
+                          label: 'Elem 3.3',
+                          id: 'tab-f-3-3'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1400]:
```suggestion
                          id: 'tab-f-2-3-3'
```

File: src/website/views/components/tabs/_examples.pug [hunk 18]
```diff
@@ -1172,42 +1278,144 @@ h2 Examples
...
+                        }
+                      ]
+                    },
+                    {
+                      label: 'Elem 4',
+                      id: 'tab-f-4'
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:1406]:
```suggestion
                      id: 'tab-f-2-4'
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/website/views/components/tabs/_examples.pug [hunk 19]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                  label: 'Element 2',
+                  id: 'tab-h-2',
+                  children: [
+                    {
+                      label: 'Elem 1',
+                      id: 'tab-h-2-1'
+                    },
+                    {
+                      label: 'Elem 2',
+                      id: 'tab-h-2'
```
Comment (jllr) [inline src/website/views/components/tabs/_examples.pug:1189]:
```suggestion
                      id: 'tab-h-2-2'
```

File: src/website/views/components/tabs/_examples.pug [hunk 20]
```diff
@@ -1075,46 +1075,152 @@ h2 Examples
...
+                    },
+                    {
+                      label: 'Elem 3 more',
+                      id: 'tab-h-3',
```
Comment (jllr) [inline src/website/views/components/tabs/_examples.pug:1193]:
```suggestion
                      id: 'tab-h-2-3',
```

**Review by jllr — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**